cocotb/config.py

```diff
@@ -107,6 +153,19 @@ def __call__(self, parser, namespace, values, option_string=None):
         parser.exit()
 
 
+class PrintFuncAction(argparse.Action):
+    def __init__(self, option_strings, dest, function=None, **kwargs):
+        super(PrintFuncAction, self).__init__(option_strings, dest, **kwargs)
```

Comment:
Prefer no arguments in Python 3.
```suggestion
        super().__init__(option_strings, dest, **kwargs)
```